src/common/components/htmlToReact/replaceComponents/IframeForEmbeddedMedia.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+/**
+ * An iframe HTML-element to securely render
+ * a sandboxed version of an embedded video media player, e.g. from the CMS.
+ * */
+export default function IframeForEmbeddedMedia(
+  props: React.PropsWithChildren<React.IframeHTMLAttributes<HTMLIFrameElement>>,
+) {
+  return (
+    // eslint-disable-next-line jsx-a11y/iframe-has-title
+    <iframe
+      {...props}
+      loading="lazy"
+      sandbox="allow-scripts allow-same-origin allow-popups allow-presentation"
```

Comment:
1. Without the "allow-presentation", one threw a warning.
2. "allow-popups" was mentioned in multiple examples related to the Youtube, but I couldn't find the feature that it was needed for. I thought it would be still ok to have here.
3. "allow-scripts allow-same-origin" is the insecure part of this iframe thing, but unfortunately it's also the mandatory part.

Reply:
Ok, just wanted to raise this question to ensure security has been thought of.